docs/content/latest/api/ysql/commands/ddl_create_table.md

```diff
@@ -223,16 +229,15 @@ yugabyte=# CREATE TABLE translations(message_id int UNIQUE,
                                      message_txt text);
 ```
 
-### Create table specifying number of tablets
+### Create a table specifying the number of tablets
 
-You can use the `CREATE TABLE` statement with the `SPLIT INTO` clause to specify the number of tablets for the table.
-This is useful for two data center (2DC) deployments that require identical number of tablets on both clusters.
+To specify the number of tablets for a table, you can use the `CREATE TABLE` statement with the [`SPLIT INTO`](#split-into) clause.
 
 ```postgresql
 yugabyte=# CREATE TABLE tracking (id int PRIMARY KEY) SPLIT (INTO 10 TABLETS);
```

Comment:
Please change to `SPLIT INTO 10 TABLETS`.
We have made parenthesis optional and are trying to move away from that.

Reply:
Got it -- fixed.